src/main/java/CarRacing/Domain/Car.java

```diff
@@ -0,0 +1,49 @@
+package CarRacing.Domain;
+
+import java.util.Objects;
+
+public class Car {
+    private static final int MOVABLE_RPM_MIN = 4;
+
+    private final Name name;
+    private Position position = Position.valueOf(0);
```

Comment:
```suggestion
    private static final int MOVABLE_RPM_MIN = 4;

    private final Name name;
    
    private Position position = Position.valueOf(0);
```

다른 키워드끼리 분리하는게 가독성이 좋습니다!